src/Debugging/Microsoft.DiaSymReader.PortablePdb.Tests/project.json

```diff
@@ -1,8 +1,4 @@
 {
-  "supports": {
```

Comment:
It verifies that the packages you reference actually will run on the stated platforms. Modulo bugs, we should have these in there.